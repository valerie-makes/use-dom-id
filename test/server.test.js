import React from "react";
import MyApp from "./__MyApp";

import { renderToString } from "react-dom/server";

describe("server-side rendering", () => {
  it("should render correctly", () => {
    const result = renderToString(<MyApp />);
    expect(result).toMatchSnapshot();
  });
});
