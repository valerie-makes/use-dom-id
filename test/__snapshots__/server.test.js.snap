// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`server-side rendering should render correctly 1`] = `"<form data-reactroot=\\"\\"><label for=\\"id-server-0\\">Full Name</label><input type=\\"text\\" id=\\"id-server-0\\" data-ssr=\\"true\\"/><label for=\\"id-server-1\\">Email Address</label><input type=\\"email\\" id=\\"id-server-1\\" data-ssr=\\"true\\"/></form>"`;
